pandas/core/generic.py

```diff
@@ -6632,6 +6632,13 @@ def groupby(self, by=None, axis=0, level=None, as_index=True, sort=True,
         squeeze : boolean, default False
             reduce the dimensionality of the return type if possible,
             otherwise return a consistent type
+        observed : boolean, default None
+            if True: only show observed values for categorical groupers.
```

Comment:
capital If (below as well)

Reply:
Also, can you start this explanation with noting this keyword is only when grouping by categorical values?